sonar-plugin/pom.xml

```diff
@@ -3,32 +3,32 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <groupId>io.ecocode</groupId>
-    <artifactId>ecocode-javascript-plugin</artifactId>
+    <groupId>org.green-code-initiative</groupId>
+    <artifactId>creedengo-javascript-plugin</artifactId>
     <version>1.5.1-SNAPSHOT</version>
```

Comment:
for me the version must be 2.0.0 now

Reply:
Yes I agree! This value will be updated automatically by the GitHub workflow that releases the plugin.